### DISCUSSION TITLE
Poster Template Powerpoint

### DISCUSSION THREAD

**Discussion by jonmatthis** [Biol2299-Matthis-Fall2022/Biol2299-Matthis-Fall2022#60] [General]
[BIOL2299 Poster Template_S22.pptx](https://github.com/Biol2299-Matthis-Fall2022/Biol2299-Matthis-Fall2022/files/9904152/BIOL2299.Poster.Template_S22.pptx)

**Comment by Julianlee24** [comment]
https://docs.google.com/presentation/d/1rvQNmtdiN81_f83-UkDDP6rauHLYBACX_YTAZP7niCQ/edit?usp=sharing

**Reply by Robinsbank** [reply]
how very nasty
